src/mustache_core.js

```diff
@@ -206,7 +206,9 @@ var core = {
 					}
 				}
 				// Look up partials in options first.
-				var partial = options.peek("partials." + localPartialName), renderer;
+				var partial = options.peek("partials." + localPartialName);
+				partial = partial || ( options.inlinePartials && options.inlinePartials[ localPartialName ] );
```

Comment:
So partials created with `registerPartial` or otherwise will take precedence over inline partials?

Would it be helpful to provide a dev warning if a partial and a named partial exist with the same name?